design/src/high-level/analysis-graphics.md

```diff
@@ -0,0 +1,77 @@
+# High-Level Analysis: Graphics
+
+## Case Studies
+
+### Terra Nil
+
+[A reverse civilization builder solar punk game](https://store.steampowered.com/app/1593030/Terra_Nil/) where the player is tasked with making a barren planet bustling with life. The player builds machines to purify water, soil, and air to breathe life back into the game world.
+
+- A _2D world_ with an _isometric camera with constrained motion_.
+
+- Animals and some player buildings' _2D assets_ have _outlines drawn with, and cell shading using, shaders_ to add definition to assets.
+
+- Various animals have _'random' animations_ that add a sense of life to the game.  
+
+- The color palette is _vivid colors for greenery_, _muted colors for wasteland_, and _soft hues of bold colors for buildings_ which helps fit the game's theme of _rejuvenating nature_.
+
+- _Textures_ give the assets _varying levels of depth_.
+
+- _Ambient lighting_ gives depth to the world, _casts shadows_, and helps _ambient clouds add texture/depth_.
+
+- Assets have a _pastoral, hand drawn aesthetic_ with mild _cell shading_.
+
+### Warcraft 3
+
+### Factorio
+
+### Minecraft
+
+### Dorfromantik
+
+### Frostpunk
+
+### Rim World
+
+### Breath of the Wild
+
+### Don't Starve Together
+
+### Paper Mario
+
+### Nova Draft
+
+### Paper Sorcerer
+
+### Journey
+
+### N++
+
+[N++](https://store.steampowered.com/app/230270/N_NPLUSPLUS/) is a fast-paced, momentum-based platformer about darting around obstacles, narrowly evading enemies and collecting gold in a beautiful minimalist landscape.
+
+- A _2D world_ with an _orthogonal, fixed camera_.
+
+### Heads Run
+
+### Jet Set Radio Future
+
+### Okami
+
+### Everything
+
+The game [_Everything_](https://store.steampowered.com/app/582270/Everything/) has the player control a collective consciousness that is a part of everything in the universe. The player is tasked with exploring our universe as a disembodied consciousness that can control anything in the (game's) universe as philosophy lectures play in the background. Built with Unity.
+
+- A _3D world_ with a _third-person perspective camera with full rotational freedom_ to allow the player to view every angle of the world.
+
+- The creator chose _low-poly models_ as assets, with the creative choice _not to use animations_ for movement but to instead move the full model in a uniform way.
```

Comment:
Can we add a gif, or a link to a timestamp in a youtube video that clearly demonstrates this effect?

Reply:
went ahead and did so! I may add a few more/swap some out if I find better ones.